### PR TITLE
Component callouts

### PR DESCRIPTION
**Pull request by Demotive**
Create basic callouts "See your GP", "Call 111", "Go to A+E"

### CODE REVIEW

File: assets/stylesheets/components/_global-header.scss [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 .global-header--inner {
   @extend %content-block;
-  padding: 24px;
+  padding: ($baseline-grid-unit * 4);
```
Comment (teneightfive) [inline assets/stylesheets/components/_global-header.scss]: Just having had a look at the layout and we should possibly handle left/right padding within either the `%content-block` mixin or as part of a grid.

Comment (teneightfive) [inline assets/stylesheets/components/_global-header.scss]: That way we keep it consistent all the way down the page

Comment (teneightfive) [inline assets/stylesheets/components/_global-header.scss]: In fact, it's actually already being looked at as part of #27 (https://github.com/nhsalpha/betahealth/pull/27/files#diff-258a86056b2ecc311b8da35def260643R7)

Comment (Demotive) [inline assets/stylesheets/components/_global-header.scss]: Yes, good point. TBH I was going to suggest we deal with this as part of a grid PR or similar?

Comment (Demotive) [inline assets/stylesheets/components/_global-header.scss]: Basically this hypothetical "grid PR" would rework %content-block anyway...

Comment (teneightfive) [inline assets/stylesheets/components/_global-header.scss]: Yep, agreed. I'd drop this change for now though as it's already been merged with #27. 

Maybe add the media query change to the top/bottom padding with this though.

Comment (Demotive) [inline assets/stylesheets/components/_global-header.scss]: 👍

**Pull request merged**